.github/container/create-distribution.sh

```diff
@@ -5,35 +5,52 @@
 usage() {
 cat <<EOF
 Usage: $0 [OPTION]...
-  -d, --dir=PATH         Path of installed upstream base library. Defaults to /opt/t5x
-  -e, --extra-dir=PATH   Path to an additional git mirror of the base library to retrieve git-refs to patch upstream.
-                           Should be a fork/mirror of the repo passed to -d/--dir. Helpful if you want to test your
-                           patch before PR-ing against upstream
-  -h, --help             Print usage.
-  -m, --mirror-url=URL   Git url of a mirror/fork of the upstream repo located at --dir
-  -p, --patchlist=PATH   Path to patchlist.txt with feature PRs
-  -r, --ref=REF          Git commit hash or tag name that specifies the base of the t5x distribution. Defaults to main (not origin/main)
+  -c, --clean               [Optional] If set, will clean the patch dir. Default is not to clean
+  -h, --help                [Optional] Print usage.
+  -m, --manifest=PATH       Path to the manifest. Updates it in-place
+  -o, --override_dir=PATH   [Optional] Use this if there is a custom location of the upstream clone. If not specified, uses /opt/\${PACKAGE}
+  -p, --package=KEY         The package name in the manifest to use, e.g., t5x, paxml
+  -s, --skip-apply          [Optional] If provided, will only create patches, update manifest, and skip applying. When not provided, applies local patches.
 
-Relationship between --dir, --extra-dir, and --mirror-url repo args:
-  --dir: The upstream repo, locally cloned
-  --mirror-url: A mirror of the upstream repo
-  --extra-dir: A locally cloned mirror of the upstream repo. Helpful to incorporate changes from private repos.
+--------------
 
-Patches in the --patchlist will be applied from the repos above according to the following rules:
+This script has two modes of operation:
```

Comment:
Could you please add a general description of the purpose of thus script? Like, ""this script is for updating manifest with patches and apply patches to the closed repos".